articles/urls.py

```diff
@@ -5,13 +5,16 @@
 
 from articles import views
 
-router = SimpleRouter()
-router.register(
+v0_router = SimpleRouter()
+v0_router.register(
     r"articles",
     views.ArticleViewSet,
     basename="articles",
 )
 
+app_name = "articles"
 urlpatterns = [
-    re_path(r"^api/v1/", include(router.urls)),
+    # TODO(Chris Chudzicki): Change this to version v0 when # noqa: FIX002
+    #  https://github.com/mitodl/mit-open/issues/269 is finished
+    re_path(r"^api/v0/", include((v0_router.urls, "v1"))),
```

Comment:
This API is something we're using for internal experimentation with CKEditor right now. It is not exposed on the public site. IMO, it should be a v0 API. However, a staff-only portion of the frontend is using this API via generated client, and the client is generated only from the v1 specification. This will change when client generation moves to https://github.com/mitodl/open-api-clients. Rather than changing the existing, to-be-removed client generation in MIT Open, I think it makes sense to keep this as v1 for now, and move it to v0 once https://github.com/mitodl/open-api-clients is ready.

Reply:
Some of this was supposed to apply to a different endpoint, sorry (ckeditor - in v0 schema but with v1 url)

Reply:
articles is in v1 schema but has a v0 url - confusing